### DISCUSSION TITLE
AnalyticalTable export to excel

### DISCUSSION THREAD

**Discussion by DrMalchev** [UI5/webcomponents-react#6411] [Q&A]
I don't see a build in way to export all or selected rows to excel (csv) in AnalyticalTable. Can you please guide me if there is a preffered way to do so. Right now we use a generic react library, but if there is something more UI5 oriented I will be happy to use it.

Thanks for your help

**Comment by Lukas742** [comment]
Hi @DrMalchev
As far as I know, there are no SAP-specific recommendations regarding converting JS data structures to excel.
